server/routers/project.userfield.router.js

```diff
@@ -3,35 +3,39 @@ const getDB = require ('../database/bootstrap.database.js');
 
 const projectUserFieldRouter = express.Router();
 
-projectUserFieldRouter.post('project/create/userfield', (req, res) => {
+projectUserFieldRouter.post('project/create/userfieldid', (req, res) => {
```

Comment:
In this url, `userfieldid` should just be `userfield` because when creating a new user field, there is no idea to reference yet.